ship.html

```diff
@@ -13,7 +13,7 @@
                 </div>
                 <div class="section3">
                     <div class="section4">
-                        <a class="section5" href="home.html">Home</a>&nbsp;&nbsp;&nbsp;
+                        <a class="section5" href="index.html">Home</a>&nbsp;&nbsp;&nbsp;
```

Comment:
It is better to use a div and give it a margin o auto and text-align center to get the same effect.

Reply:
am not getting you on this... am I supposive to put the a href tag in  the div tag oba?